src/main/java/ch/njol/skript/events/EvtEntityBlockChange.java

```diff
@@ -89,15 +91,15 @@ public boolean init(Literal<?>[] args, int matchedPattern, ParseResult parser) {
 	}
 	
 	@Override
-	public boolean check(Event e) {
-		if (!(e instanceof EntityChangeBlockEvent))
+	public boolean check(Event event) {
+		if (!(event instanceof EntityChangeBlockEvent))
 			return false;
-		return event.checker.check((EntityChangeBlockEvent) e);
+		return this.event.checker.check((EntityChangeBlockEvent) event);
 	}
 	
 	@Override
-	public String toString(@Nullable Event e, boolean debug) {
-		return event.name().toLowerCase(Locale.ENGLISH).replace('_', ' ');
+	public String toString(@Nullable Event event, boolean debug) {
+		return this.event.name().toLowerCase(Locale.ENGLISH).replace('_', ' ');
```

Comment:
Same as above

Reply:
`private ChangeEvent event` at L85 too, these were referencing to the ChangeEvent enum.